src/main/java/org/openrewrite/java/ReloadableJava8ParserVisitor.java

```diff
@@ -446,11 +446,19 @@ public J visitCompilationUnit(CompilationUnitTree node, Space fmt) {
 
         endPosTable = cu.endPositions;
 
+        Map<Integer, JCAnnotation> annotationPosTable = new HashMap<>();
+        if (node.getPackageAnnotations() != null) {
+            for (AnnotationTree annotationNode : node.getPackageAnnotations()) {
+                JCAnnotation annotation = (JCAnnotation) annotationNode;
+                annotationPosTable.put(annotation.pos, annotation);
+            }
+        }
+        List<J.Annotation> kindAnnotations = annotationPosTable.isEmpty() ? null : collectAnnotations(annotationPosTable);
```

Comment:
I don't think "kindAnnotations" is what this should be called. Looks like a copy+pasted name 
"Kind" is the type that differentiates between class/interface/enum. Packages don't have such a concept.

Reply:
You're correct I will change the name. Thanks!